### PR TITLE
Ensure the active bolus screen alert does not change until cleared

### PR DESCRIPTION
**Pull request by mpangburn**
SwiftUI blows up if the active alert changes out from under it.

### CODE REVIEW

File: Loop/View Models/BolusEntryViewModel.swift [hunk 1]
```diff
@@ -179,35 +179,35 @@ final class BolusEntryViewModel: ObservableObject {
...
         if let manualGlucoseSample = manualGlucoseSample {
             guard Self.validManualGlucoseEntryRange.contains(manualGlucoseSample.quantity) else {
-                activeAlert = .manualGlucoseEntryOutOfAcceptableRange
+                presentAlert(.manualGlucoseEntryOutOfAcceptableRange)
                 return
             }
 
             isInitiatingSaveOrBolus = true
-            dataManager.loopManager.glucoseStore.addGlucose(manualGlucoseSample) { result in
+            dataManager.loopManager.addGlucose([manualGlucoseSample]) { result in
```
Comment (mpangburn) [inline Loop/View Models/BolusEntryViewModel.swift:203]: (I realized that LoopDataManager has an API that already calls in to the GlucoseStore API, better to just use that one. Tiny change so I'm tossing it in here.)

File: Loop/View Models/BolusEntryViewModel.swift [hunk 2]
```diff
@@ -279,6 +279,17 @@ final class BolusEntryViewModel: ObservableObject {
         )
     }
 
+    private func presentAlert(_ alert: Alert) {
+        dispatchPrecondition(condition: .onQueue(.main))
+
+        // As of iOS 13.6 / Xcode 11.6, swapping out an alert while one is active crashes SwiftUI.
+        guard activeAlert == nil else {
```
Comment (rickpasetto) [inline Loop/View Models/BolusEntryViewModel.swift:286]: oy...this is only in 13.6? :\

**Review by rickpasetto — APPROVED** [review]
LGTM 🍀

File: Loop/View Models/BolusEntryViewModel.swift (hunk 2 above)
Comment (mpangburn) [reply Loop/View Models/BolusEntryViewModel.swift:286]: Probably prior as well, I left the comment just to note "this was the latest version I've checked on"

**Pull request merged**